ampersand-dom-bindings.js

```diff
@@ -35,7 +40,7 @@ function switchHandler(binding, el, value) {
     for (var item in binding.cases) {
         var curValue = binding.cases[item];
         if (value !== item && curValue !== showValue) {
-            getMatches(el, curValue).forEach(function (match) {
+            getMatches(el, curValue, binding.firstMatchOnly).forEach(function (match) {
```

Comment:
I think you're right, I missed it.

TBH I didn't fully understand the case and, since the tests came out ok, I thought it was ok. I'll review the de case and push changes ASAP. Thanks!